app/controllers/subject_viewer.coffee

```diff
@@ -69,13 +81,17 @@ class SubjectViewer extends Controller
     if @classification
       @classification.bind 'change', @onClassificationChange
       @classification.bind 'add-species', @onClassificationAddSpecies
-
-      @html template @classification
-
-      @active = Math.floor @classification.subject.location.standard.length / 2
-      @activate @active
-
-      @onClassificationChange()
+      deferred = Experiments.getCohortRetriever()
```

Comment:
I find that confusing - you're saying that the result of getCohort should be both a promise that you can attach thens to, but *also* evaluate to the string value of response.cohort? Surely it needs to be returned as a Promise, not a string, so that the caller can still add thens if they want to?